core/primitives/src/stateless_validation.rs

```diff
@@ -16,8 +16,13 @@ use near_primitives_core::hash::CryptoHash;
 use near_primitives_core::types::{AccountId, Balance, BlockHeight, ShardId};
 use near_primitives_core::version::PROTOCOL_VERSION;
 
-// The value here is the same as NETWORK_MESSAGE_MAX_SIZE_BYTES.
-pub const MAX_CHUNK_STATE_WITNESS_SIZE: ByteSize = ByteSize::mib(512);
+/// Represents max allowed size of the compressed state witness,
+/// corresponds to EncodedChunkStateWitness struct size.
+pub const MAX_COMPRESSED_STATE_WITNESS_SIZE: ByteSize = ByteSize::mib(32);
+
+/// Represents max allowed size of the raw (not compressed) state witness,
+/// corresponds to the size of borsh-serialized ChunkStateWitness.
```

Comment:
nit. I think MAX_UNCOMPRESSED_STATE_WITNESS_SIZE could be clearer. or maybe MAX_STATE_WITNESS_SIZE? "raw" is ambiguous.

Reply:
agree, done in f91fc2cd6